libr/bin/bin.c

```diff
@@ -848,7 +857,9 @@ R_API int r_bin_load_io_at_offset_as_sz (RBin *bin, RIODesc *desc, ut64 baseaddr
 	if (loadaddr == UT64_MAX) {
 		loadaddr = 0;
 	}
-	file_sz = iob->desc_size (io, desc);
+
+	buf_bytes = NULL;
+	file_sz = iob->desc_size (desc);
```

Comment:
I gave it RIO, bc I aim to extend the io api with some wrappers that feel more POSIX-like